tests/backup/backup_orphaned_test.go

```diff
@@ -1000,8 +1002,10 @@ var _ = Describe("{DeleteObjectsByMultipleUsersFromNewAdmin}", func() {
 			})
 			Step(fmt.Sprintf("Delete user %s backups from the admin", user), func() {
 				log.InfoD(fmt.Sprintf("Deleting user %s backups from the admin", user))
+				nonAdminCtx, err := backup.GetNonAdminCtx(user, commonPassword)
+				log.FailOnError(err, "failed to fetch user %s ctx", user)
 				for backupName := range userBackupMap[user] {
-					backupUid, err := Inst().Backup.GetBackupUID(newAdminCtx, backupName, orgID)
+					backupUid, err := Inst().Backup.GetBackupUID(nonAdminCtx, backupName, orgID)
```

Comment:
Is this necessary since the backup names will now be unique?

Reply:
Done